WordPress/Classes/Models/WPAccount+Lookup.swift

```diff
@@ -0,0 +1,157 @@
+import CoreData
+
+public extension WPAccount {
+
+    // MARK: - Relationship Lookups
+
+    /// Is this `WPAccount` object the default WordPress.com account?
+    ///
+    @objc
+    var isDefaultWordPressComAccount: Bool {
+        guard let uuid = UserSettings.defaultDotComUUID, uuid.count > 0 else {
+            return false
+        }
+
+        return self.uuid == uuid
+    }
+
+    /// Does this `WPAccount` object have any associated blogs?
+    ///
+    @objc
+    var hasBlogs: Bool {
+        return !blogs.isEmpty
+    }
+
+    // MARK: - Object Lookups
+
+    /// Returns the default WordPress.com account, if one exists
+    ///
+    /// The default WordPress.com account is the one used for Reader and Notifications.
+    ///
+    static func lookupDefaultWordPressComAccount(in context: NSManagedObjectContext) throws -> WPAccount? {
+        guard let uuid = UserSettings.defaultDotComUUID, uuid.count > 0 else {
+            // No account, or no default account set. Clear the defaults key.
+            UserSettings.defaultDotComUUID = nil
+            return nil
+        }
+
+        return try lookup(withUUIDString: uuid, in: context)
+    }
+
+    /// Does a default WordPress.com account exist?
+    ///
+    /// The default WordPress.com account is the one used for Reader and Notifications.
+    ///
+    static func lookupHasDefaultWordPressComAccount(in context: NSManagedObjectContext) throws -> Bool {
```

Comment:
Could maybe just be `hasDefaultWordPressComAccount`?   It might just be me but when prefixed with `lookup` I could see this mistaken for the actual look up method at a glance. `lookup` also makes me thing an account instance could be returned where `has` implies a boolean.  Thoughts?

Reply:
This makes sense to me, and helps distinguish the two methods. Alternatively, `defaultWordPressComAccountExists(in:)`?